docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,122 @@
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based application.
+* RHEL 7 installed.
+* OpenJDK 8 installed.
+* Maven 3.5 or later installed.
+
+
+.Procedure
+
+. Ensure that you specify the link:https://maven.repository.redhat.com/[Red Hat Maven repository] in the `pom.xml` of your project.
+Also ensure that the {parameter-runtime-name} Maven plugin is configured:
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project ... >
+  ...
+ifdef::built-for-spring-boot,built-for-spring-boot-1-5[]
+  <!-- Specify target artifact type for the repackage goal. -->
+  <packaging>jar</packaging>
+endif::[]
+  ...
+  <repositories>
+    <repository>
+      <id>{red-hat-ga-repo-id}</id>
+      <name>{red-hat-ga-repo-name}</name>
+      <url>{red-hat-ga-repo-url}</url>
+    </repository>
+  </repositories>
+  ...
+  <pluginRepositories>
+      <pluginRepository>
+        <id>{red-hat-ga-repo-id}</id>
+        <name>{red-hat-ga-repo-name}</name>
+        <url>{red-hat-ga-repo-url}</url>
+      </pluginRepository>
+    </pluginRepositories>
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>{parameter-group-name-runtime-plugin}</groupId>
+        <artifactId>{parameter-artifact-name-runtime-plugin}</artifactId>
+        <version>{parameter-version-runtime-plugin}</version>
+ifdef::built-for-vertx[]
+        <executions>
+          <execution>
+            <id>vmp</id>
+            <goals>
+              <goal>initialize</goal>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+endif::[]
+ifdef::built-for-thorntail[]
+        <executions>
+          <execution>
+            <id>package</id>
+            <goals>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+endif::[]
+ifdef::built-for-spring-boot,built-for-spring-boot-1-5[]
+        <executions>
+          <execution>
+              <goals>
+               <goal>repackage</goal>
+             </goals>
+          </execution>
+        </executions>
+endif::[]
+      </plugin>
+    </plugins>
+  </build>
+  ...
+</project>
+----
+
+. Package your application.
+Consider packaging your application as a fat jar to include all of the necessary dependencies.
+ifdef::built-for-vertx[The Vert.x Maven Plugin packages applications as fat jars link:https://reactiverse.io/vertx-maven-plugin/#introduction[by default].]
+ifdef::built-for-thorntail[The Thorntail Maven Plugin lets you package applications as link:https://docs.thorntail.io/2.3.0.Final/#uberjar_thorntail[_uberjars_] or link:https://docs.thorntail.io/2.3.0.Final/#creating-a-hollow-jar_thorntail[hollow JARs].]
```

Comment:
Please link to 2.4.0.Final.

Reply:
Also applications are _not_ packaged as hollow JARs. Best to not mention the hollow JAR at all in this context.